etest/__init__.py

```diff
@@ -88,6 +92,8 @@ def _(check):
 
         threading.Thread(target=_, args=(check,)).start()
 
+    qemu.exit()
```

Comment:
Does exit require start to have been called?  This would feel much better with a context manager.

Reply:
It does. It has caused problems for me, its fixed in the next commit.